uber/utils.py

```diff
@@ -77,6 +77,30 @@ def localize_datetime(dt):
     return dt.replace(tzinfo=UTC).astimezone(c.EVENT_TIMEZONE)
 
 
+def ceil_datetime(dt, delta):
+    """Only works in Python 3"""
+    dt_min = datetime.min.replace(tzinfo=dt.tzinfo)
+    dt += (dt_min - dt) % delta
+    return dt
+
+
+def floor_datetime(dt, delta):
+    """Only works in Python 3"""
+    dt_min = datetime.min.replace(tzinfo=dt.tzinfo)
+    dt -= (dt - dt_min) % delta
+    return dt
+
+
+def noon_datetime(dt):
```

Comment:
It'd be nice if this and the `evening_datetime` function was just a single utility function that took an hour as an argument, since they're so similar.

Reply:
Yep, strong agree!